corehq/apps/cloudcare/static/cloudcare/js/formplayer/users/models.js

```diff
@@ -11,6 +11,17 @@ FormplayerFrontend.module("Users.Models", function(Models, FormplayerFrontend, B
             }.bind(this));
         },
 
+        isMobileWorker: function() {
+            return this.username.endsWith('commcarehq.org');
```

Comment:
not a blocker, just wondering if you think it's worth grabbing this from `settings.HQ_ACCOUNT_ROOT` as is done here https://github.com/dimagi/commcare-hq/blob/24b3679279ad19093b8aca28b765eff2c5c5a8c4/corehq/apps/users/util.py#L29